rfcs/xxxx-template.md

```diff
@@ -0,0 +1,42 @@
+- Start Date: (2023-04-18)
+- RFC PR:
+
+# Summary
+Implement a category dropdown in the admin UI for both Collection & Single types.
+
+# Motivation
+The motivation behind this proposal is to simplify content organization and improve the user experience while managing content in the Strapi admin interface. Currently, users have to navigate through a long list of Collection and Single types, which can be time-consuming and cumbersome, especially when dealing with numerous content types. By introducing a category dropdown, users can easily group and filter their content types, making content management more efficient and user-friendly.
+
+# Detailed design
+The proposed solution includes the following design changes:
+
+1. Add a category dropdown in the admin UI for both Collection & Single types, enabling users to filter content types based on the selected category.
+2. Allow users to create, edit, and delete categories from the admin UI.
```

Comment:
if we want to make it less complex we would could just use a string in the content-type for this 

create: you create type in the new name or select from dropdown from all existing categories or leave it empty for uncategorized.

delete: after you delete or change the last item in the category the category will be auto deleted since there are no content-types left with it in info.Category 

aka we add to the parser of the content-types on startup to create the categories and save them in memory 

this means that this paser does all the heavy lifting since on restart it re generates all the categories